gnomad/utils/constraint.py

```diff
@@ -53,35 +61,45 @@ def count_variants_by_group(
     """
     Count number of observed or possible variants by context, ref, alt, and optionally methylation_level.
 
-    Performs variant count aggregations based on specified criteria (`count_singletons`, `count_downsamplings`, and
-    `max_af`), and grouped by: 'context', 'ref', 'alt', 'methylation_level' (optional), and all annotations provided
-    in `additional_grouping`.
+    Performs variant count aggregations based on specified criteria
+    (`count_singletons`, `count_downsamplings`, and `max_af`), and grouped by:
+    'context', 'ref', 'alt', 'methylation_level' (optional), and all annotations
```

Comment:
Yes! Maybe it will be useful for v4 so we kept this.

Reply:
wouldn't we still want to separate variants by methylation level? otherwise agree it's good to keep the code flexible

Reply:
I remember we decided to keep this because we want to it to be generic in `gnomad_method`. It will also be used in building mutation rate table as well (see [here](https://github.com/broadinstitute/gnomad_lof/blob/e09d9fdc13d43c8ac996c93735ed206ba41ed473/constraint_utils/constraint_basics.py#L371)).